### ISSUE TITLE
Add Flash Share tab

### ISSUE THREAD

**Issue by bergware** [limetech/dynamix#156]
The comment on the forum made me rethink the shares approach for Flash.

I have added a third tab called "Flash Share" under Shares. This tab is unconditional and is shown when the array is started or stopped. It is also treated as a separate entity, meaning the N/P buttons are not available under flash, while the N/P buttons under Disk Shares have Flash excluded.

Following this approach we can truely say that under *Main* all device settings are given, while under *Shares* all share settings are given.

The rearranging is pretty easy to do, and if you agree or like to see the result, I can put it on github.

Let me know.

**Comment by limetech** [comment]
Yes I went back and forth on this.  Jon likes it where it would show up under Shares too, so I will go ahead and merge this.

Also, we fixed up the code to correctly deal with share names that have an apostrophe (single quote).  For example in ShareList.page we see this:

```
$cmd=urlencode("$docroot/webGui/scripts/share_size " . escapeshellarg($name) . " " . escapeshellarg("/var/local/emhttp/$name.ssz1"));
```

But there is a bug remaining in this area with the N/P buttons:

1 - Go to Shares page on main tab
2 - Create a share using a name with an apostrophe (single quote)
3 - Click Shares again from the main taskbar
4 - Click any share name
5 - Click the > or < symbols to cycle between shares
6 - When you reach the share with the apostrophe, you will get an error message reported that says the share has been deleted (it hasn't, but you just can't get to it).
